ocs_ci/ocs/ui/page_objects/virtualmachine_ui.py

```
@@ -432,6 +480,89 @@ def check_grace_period_and_confirm_delete(self):
         self.do_click(delete_btn, enable_screenshot=True)
         logger.info("Clicked Delete in confirmation modal")
 
+    def click_virtual_machines_tab_and_open_vm(self, vm_name, namespace):
```

Ruff (0.16.0)

[warning] Unused method argument: `namespace`

(ARG002)